src/Controls/samples/Controls.Sample.UITests/Controls.Sample.UITests.csproj

```diff
@@ -48,6 +48,10 @@
     <MauiSplashScreen Include="Resources\Splash\splash.svg" Color="#FFFFFF" BaseSize="168,208" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\Fonts\OpenSans-Regular.ttf" />
```

Comment:
Maybe this should just be `<MauiFont>` ?

Reply:
Non-packaged app fonts won't work if they're set to `MauiFont`

Reply:
should be fixed now - we merged the win2d updates.

But it is fine if we want this to run on older branches.